spec/e2e/features/short_form/sign_in_while_applying.feature

```diff
@@ -6,6 +6,7 @@ Feature: Sign-in while filling out application
     Scenario: Setting up an account for sign in while applying tests
         # Birth date indicates < 65 years old for senior building tests
         Given I have a confirmed account for "Alice Walker" with birth date "1/1/2000"
+        When I wait "5" seconds
```

Comment:
@callaghanc so this error was just being caused by a timeout. In general, the tests were very slow on my computer. Do you have any ideas for how to add a more consistent wait until done on the account confirmation?